src/Menge/MengeCore/SceneGraph/Context.cpp

```diff
@@ -3,7 +3,7 @@
 License
 
 Menge
-Copyright � and trademark � 2012-14 University of North Carolina at Chapel Hill. 
+Copyright � and trademark � 2012-14 University of North Carolina at Chapel Hill. 
```

Comment:
here again. Can you change this? Or should I? 
This will repeat in a couple of files later...